src/pages/Showroom/Showroom.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import ShowroomItem from './ShowroomItem/ShowroomItem';
+import './Showroom.scss';
+
+const Showroom = () => {
+  const [showroomData, setShowroomData] = useState([]);
+
+  useEffect(() => {
+    fetch('http://10.58.52.225:3000/products/showRoom', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    })
+      .then(response => response.json())
+      .then(result => {
+        setShowroomData(result);
+      });
+  }, []);
+
+  return (
+    <div className="showroom">
+      <div className="header-main">온라인 쇼룸</div>
+      {showroomData.map(
+        ({ id, image_url, header, descriptions, products, price }) => {
+          return (
+            <ShowroomItem
+              image_url={image_url}
+              key={id}
+              header={header}
+              descriptions={descriptions}
+              products={products}
+              price={price}
+              rightAligned={id % 2 === 0 ? true : false}
+            />
+          );
```

Comment:
```suggestion
        (list) => {
          return (
            <ShowroomItem
              key={list.id}
              rightAligned={list.id % 2 === 0 ? true : false}
              {...list}
            />
          );
```
구조분해 할당한 모든 데이터를 props로 넘기는 거라면
이렇게 넘겨주는 것이 조금 더 가독성 측면에서 깔끔해 보이네요!